eng/build.sh

```diff
@@ -325,9 +330,9 @@ BuildSolution
 if [[ "$test_core_clr" == true ]]; then
```

Comment:
Hey I am handling this separately in this PR: https://github.com/dotnet/fsharp/pull/17333
Since such a change activate previously not executed tests, some of them are failing.
Let's fix that separately.